bootstrap/013_task_queue.sql

```diff
@@ -0,0 +1,54 @@
+
+create or replace procedure task_queue.run_tasks()
+returns varchar
+language sql
+as
+begin
```

Comment:
Probably want to generate a row in internal.task_log for this "meta" task?

Reply:
sorry, would want to add in to `task_queue.run_tasks_task`, not here.

Reply:
done